### DISCUSSION TITLE
Tips in Contributing in OpenSource

### DISCUSSION THREAD

**Discussion by Jumanjigobez** [EddieHubCommunity/support#7556] [First-Timers] [answered]
Any Tips kindly?

**Comment by Panquesito7** [comment, marked as answer]
Hey there! Here are a few tips on my side:

- First off, check tips from the EddieHub community: https://github.com/EddieHubCommunity/support/tree/main/docs/tips
- Try exploring projects that fit your tech stack. You can search in [GitHub Explore](https://github.com/explore), [OSWC](https://oswc.is), and the [Good First Issue Finder](https://finder.eddiehub.io/) to find good first issues easier as well as get familiarized with the projects.
- Before creating a pull request, you can create an issue in the repository to discuss it with the community so that you don't waste your time on a feature that could be rejected.
- Search for existing issues before creating yours, as it could be a duplicate and unnecessary noise.
- If there's anything you can help with, don't hesitate, and help! Helping others is something you should _always_ consider doing. 🙂
- To get more involved in a project, check out their links, readme, and the discussions if they have any.
- If the project is on a tech stack you aren't familiar with, you can still contribute to it in the documentation and other minor fixes. Every kind of contribution is very appreciated!

There are many more tips on the internet. You can also search for that. 🙂
I hope that helped. If you need any more help, let us know!

**Reply by Jumanjigobez** [reply]
Perfect Thanks Maan, I appreciate.
